src/components/MapView/MapView.web.tsx

```diff
@@ -87,6 +89,7 @@ const MapView = forwardRef<MapViewHandle, MapViewProps>(
                         latitude: initialState.location[1],
                         zoom: initialState.zoom,
                     }}
+                    style={StyleUtils.getTextColorStyle(themeColors.shadow) as React.CSSProperties}
```

Comment:
Sorry for the delay 🙇🏼 I believe it would be better to create a new constant inside `themeColors` for this color. This way, it will always be black and independent of any color change.